src/sentry/dynamic_sampling/latest_release_booster.py

```diff
@@ -82,4 +82,4 @@ def add_boosted_release(project_id, release_id):
         release_id,
         datetime.utcnow().replace(tzinfo=UTC).timestamp(),
     )
-    redis_client.pexpire(cache_key, ONE_DAY_TIMEOUT)
+    redis_client.pexpire(cache_key, BOOSTED_RELEASE_TIMEOUT * 1000)
```

Comment:
so only 1h for expire?

Reply:
Yes for the hashset should be one hour yes 
It was a bug